### ISSUE TITLE
newline after + sign

### ISSUE THREAD

**Issue by yixiangD** [yihui/formatR#100]
Hi, Is it possible to split a long ggplot pipeline into newlines whenever there is a + sign.
Many thanks!

**Comment by yihui** [comment]
No, it's currently not possible. I guess it could be implemented, but I don't have time for this task. If anyone is interested in implementing it, I can write down my ideas for the possible implementation.
